build.sbt

```diff
@@ -75,12 +75,11 @@ libraryDependencies ++= jacksonOverrides ++  Seq(
     "com.gu" %% "fapi-client-play28" % "4.0.4",
     "com.gu" %% "pan-domain-auth-play_2-8" % "4.0.0",
     "com.gu" %% "story-packages-model" % "2.2.0",
-    "com.gu" %% "thrift-serializer" % "4.0.0",
+    "com.gu" %% "thrift-serializer" % "4.0.2",
```

Comment:
This was the first version of `thrift-serializer` to support Scala 2.13, so this is the most minimal change I could make to get the project to compile. There have been many newer versions since then, the current is 5.0.7:

https://index.scala-lang.org/guardian/thrift-serializer/artifacts/thrift-serializer